public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionComplete.js

```diff
@@ -269,10 +269,13 @@ function ModalMissionComplete (svl, missionContainer, missionModel, taskContaine
 
     this.update = function (mission, neighborhood) {
         // Update the horizontal bar chart to show how much distance the user has audited
-        var unit = {units: 'miles'};
+        var measurementSystem = i18next.t('measurement-system');
+        var unit = {units: "miles"};
```

Comment:
I used dot access to get a String for the `mission.getDistance` method.  I didn't convert the variable unit to a String because in later lines there are method such as `completedLineDistance` and `completedLineDistanceAcrossAllUsersUsingPriority` that I think have to take an object as parameter.

Reply:
Ahh sorry, read this stuff incorrectly, you're right.